include/Game/Entities/GasHiba.h

```diff
@@ -119,6 +122,12 @@ struct ProperAnimator : public EnemyAnimatorBase {
 
 /////////////////////////////////////////////////////////////////
 // STATE MACHINE DEFINITIONS
+enum StateID {
```

Comment:
Would maybe consider making these `GASHIBA_State` rather than `HIBA_State` to be super clear, but really not a big deal, especially considering they're identical to HIBA anyway.

Reply:
ngl i wanna just recycle them for all hibas

Reply:
I mean, there are only three hibas, and ElecHiba has an additional state. up to you if you wanna change it, i'm not super fussed either way

Reply:
is the new state able to be tacked on as a new member of the enum?

Reply:
will have to decomp ElecHibaState to find out :p